src/app/debug/filter-side-drawer/filter-side-drawer.component.html

```diff
@@ -0,0 +1,20 @@
+<div @removeTrigger class="filter-vertical-panel" data-cy-debug="filter-side-drawer" *ngIf="shouldShowFilter">
+  <label class="filter-label">Filter</label>
+  <div class="filter-input-container">
+    @for (metadataName of metadataNames; track metadataNames) {
+      <div class="metadata-input-container">
+        <label class="metadata-input-label">{{ metadataName }}</label>
+        <input
+          class="metadata-input"
+          type="text"
+          [(ngModel)]="filters[metadataName]"
+          [title]="metadataName"
+          placeholder="Filter for {{metadataName}}"
```

Comment:
We created a convention (which is not published yet) for sting interpolation and binding:

```typescript
// Bad
<option value="{{view.key}}">
  {{ view.value }}
</option>


// Bad
<option [value]="'{{view.key}}px'">
  {{ view.value }}
</option>


// Good
<option [value]="view.key + 'px'">
  {{ view.value }}
</option>


// Good
<option value="15px">
  {{ view.value }}
</option>
```

So then your code should be:

```suggestion
          [placeholder]="'Filter for' + metadataName"
```

Reply:
alright. good to know